activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -141,6 +141,10 @@ def supports_datetime_with_precision?
         end
       end
 
+      def supports_virtual_columns?
+        !mariadb? && version >= "5.7.5"
```

Comment:
Glad to know you agreed.  Let's support virtual columns schema migrations and schema dump only for MySQL 5.7 or higher version, no MariaDB support added this time.